internal/cli/config/config.go

```diff
@@ -48,38 +49,48 @@ func Init(configPath string) {
 	Global.filePath = configPath
 }
 
+type Dac struct {
+	// outputFolder is the folder where the dac-generated files are stored
+	OutputFolder string `json:"output_folder,omitempty" yaml:"output_folder,omitempty"`
+}
+
 type Config struct {
 	RestClientConfig perseshttp.RestConfigClient `json:"rest_client_config"`
 	Project          string                      `json:"project,omitempty"`
 	RefreshToken     string                      `json:"refresh_token,omitempty"`
 	filePath         string
-	apiClient        api.ClientInterface
+	APIClient        api.ClientInterface
```

Comment:
I would avoid to expose this parameter. It's not supposed to be accessible directly, but through the Getter.
Exposing it that encourage to use it directly.